{"name": "测试用例描述：错误的密码登录失败", "status": "passed", "description": "测试描述：错误的密码登录失败", "steps": [{"name": "输入错误的密码", "status": "passed", "attachments": [{"name": "错误的密码", "source": "b2abdfee-e8ac-4d3c-a3b4-499759a71a77-attachment.attach"}], "start": 1651987999271, "stop": 1651987999272}], "attachments": [{"name": "截图", "source": "d4f4cee5-3e4a-4452-8958-b5cc51849300-attachment.png", "type": "image/png"}, {"name": "截图", "source": "c9af8ffe-1ce6-4de5-92b5-2b5da21a38f6-attachment.png", "type": "image/png"}], "start": 1651987999271, "stop": 1651988000985, "uuid": "6f886e7f-4587-4042-8d57-2bda22a073dc", "historyId": "e042998cd34bf7d92d802f115102d44e", "testCaseId": "ce17e49339328b1be4c6805d1bcb9a3f", "fullName": "test_two.Test#test_22", "labels": [{"name": "epic", "value": "xxx测试"}, {"name": "severity", "value": "critical"}, {"name": "story", "value": "登录"}, {"name": "feature", "value": "登录模块"}, {"name": "suite", "value": "test_two"}, {"name": "subSuite", "value": "Test"}, {"name": "host", "value": "MS-SLSXLICKSUSF"}, {"name": "thread", "value": "18340-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_two"}]}